project_proposal/tex/mainpage.tex

```diff
@@ -76,6 +76,20 @@ \section{Methods}
     \label{table:Methods}
 \end{table} 
 
+\subsection{Tools}
+The table below are some of the tools the group are going to need through out
+the project. If there is any missing tools not on the list, it will be added.
```

Comment:
Double negativ (missing tools not on the list).
Consider rewriting to something like:

"The list will be updated if it the group uses tools not yet on the list."